Extensions/EdgeScrollCamera.json

```diff
@@ -0,0 +1,1473 @@
+                        "False"
+                      ],
+                      "subInstructions": []
+                    },
+                    {
+                      "type": {
+                        "inverted": false,
+                        "value": "SceneVariableAsBoolean"
+                      },
+                      "parameters": [
+                        "__EdgeScrollCamera.MovingDown",
+                        "False"
+                      ],
+                      "subInstructions": []
+                    },
+                    {
+                      "type": {
+                        "inverted": false,
+                        "value": "SceneVariableAsBoolean"
+                      },
+                      "parameters": [
+                        "__EdgeScrollCamera.MovingLeft",
+                        "False"
+                      ],
+                      "subInstructions": []
+                    },
+                    {
+                      "type": {
+                        "inverted": false,
+                        "value": "SceneVariableAsBoolean"
+                      },
+                      "parameters": [
+                        "__EdgeScrollCamera.MovingRight",
+                        "True"
+                      ],
+                      "subInstructions": []
+                    }
+                  ],
+                  "actions": [
+                    {
+                      "type": {
+                        "inverted": false,
+                        "value": "SetCameraX"
+                      },
+                      "parameters": [
+                        "",
+                        "+",
+                        "GetArgumentAsNumber(\"ScrollSpeedRight\")*TimeDelta()",
+                        "GetArgumentAsString(\"Layer\")",
+                        "GetArgumentAsNumber(\"Camera\")"
+                      ],
+                      "subInstructions": []
+                    }
+                  ],
+                  "events": []
+                }
+              ],
+              "parameters": []
+            },
+            {
+              "colorB": 228,
+              "colorG": 176,
+              "colorR": 74,
+              "creationTime": 0,
+              "disabled": false,
+              "folded": false,
+              "name": "Diagonals",
+              "source": "",
+              "type": "BuiltinCommonInstructions::Group",
+              "events": [
+                {
+                  "disabled": false,
+                  "folded": false,
+                  "type": "BuiltinCommonInstructions::Comment",
+                  "color": {
+                    "b": 109,
+                    "g": 230,
+                    "r": 255,
+                    "textB": 0,
+                    "textG": 0,
+                    "textR": 0
+                  },
+                  "comment": "NOTE: Speeds are multiplied by \"0.7071\" to keep the linear speed constant",
```

Comment:
How did you find this magic value?

Reply:
Why not use `Sqrt(2)`?
sqrt(2) = 0.70710678118654752440084436210485

Reply:
> How did you find this magic value?

It is cos(45) and sin(45) and apparently also "sqrt(2)".  I used a constant to save CPU, but maybe it is not a good idea?